tests/core/pyspec/eth2spec/test/helpers/deposits.py

```diff
@@ -160,3 +164,57 @@ def prepare_state_and_deposit(spec, state, validator_index, amount, withdrawal_c
     state.eth1_data.deposit_root = root
     state.eth1_data.deposit_count = len(deposit_data_list)
     return deposit
+
+
+#
+# Run processing
+#
+
+
+def run_deposit_processing(spec, state, deposit, validator_index, valid=True, effective=True):
```

Comment:
Nothing changed here, right? just move to helpr file?

Reply:
Yes, just copy/paste.